ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/behaviorTree/action/actions/SceneActionNodeDefinition.java

```diff
@@ -2,47 +2,140 @@
 
 import behavior_msgs.msg.dds.SceneActionNodeDefinitionMessage;
 import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
 import com.fasterxml.jackson.databind.node.ObjectNode;
+import gnu.trove.list.array.TIntArrayList;
 import us.ihmc.behaviors.behaviorTree.BehaviorTreeRootNodeDefinition;
 import us.ihmc.behaviors.behaviorTree.action.ActionNodeDefinition;
 import us.ihmc.communication.crdt.CRDTBidirectionalEnumField;
+import us.ihmc.communication.crdt.CRDTBidirectionalEnumList;
+import us.ihmc.communication.crdt.CRDTBidirectionalBoolean;
+import us.ihmc.communication.crdt.CRDTBidirectionalFloat;
+import us.ihmc.communication.crdt.CRDTBidirectionalInteger;
+import us.ihmc.communication.crdt.CRDTBidirectionalIntegerList;
+import us.ihmc.communication.crdt.CRDTBidirectionalString;
+import us.ihmc.communication.crdt.CRDTBidirectionalStringList;
 import us.ihmc.perception.detections.foundationPose.IsaacROSFoundationPoseObject;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class SceneActionNodeDefinition extends ActionNodeDefinition
 {
+   private final CRDTBidirectionalString yoloModelName;
+   private final CRDTBidirectionalFloat yoloConfidenceThreshold;
+   private final CRDTBidirectionalFloat yoloMaskThreshold;
+   private final CRDTBidirectionalInteger segmentationMaskErosionRadius;
+   private final CRDTBidirectionalFloat outlierThreshold;
    private final CRDTBidirectionalEnumField<IsaacROSFoundationPoseObject> objectType;
+   private final CRDTBidirectionalBoolean useFoundationPose;
+   private final CRDTBidirectionalStringList enabledYoloModels;
+   private final CRDTBidirectionalIntegerList ignoredYoloClassIndices;
+   private final CRDTBidirectionalEnumList<IsaacROSFoundationPoseObject> enabledFoundationPoseModels;
```

Comment:
I added a ton of potentially useful fields to the scene action node. We still need to implement the RDX and executor for some of this.